msx/callbnk.asm

```diff
@@ -6,6 +6,13 @@
 ;
 ; Labels are defined at the main file, not here, so that this file can be included twice.
 
+; crisag - Key Changes:
+; - Bank switching now uses Konami SCC mapper addresses:
+;   5000h-57FFh for 4000h-5FFFh region
+;   7000h-77FFh for 6000h-7FFFh region
+; - Fixed banks 0 (0000h-3FFFh) and 2 (8000h-FFFFh) are assumed.
+; - Ensures correct switching before and after the routine call.

```

Comment:
I understand that this applies to your own fork of the code but not to the changes in this pull request, right? In that case it's better to remove these comments.